### DISCUSSION TITLE
Notification on new user / default group

### DISCUSSION THREAD

**Discussion by amb1** [cmintey/wishlist#186] [Ideas]
Hey, i would love to see some notification as an admin (e-mail, pop up in the frontend) as soon as a new user has registered. Currently there is no indicator if/when someone accepted an invitation or has registered by themselves.

Also is there currently a way to ask for access to a certain group?

A nother good setting would be, to be able to set a default group when inviting someone so that the invite token is somehow bound the a groupId when they register

**Comment by cmintey** [comment]
Hi! I can see a notification being helpful, I'll consider it.

There isn't a way to request access to a group through the app, another thing I will consider adding.

When you create an invite token, there are two ways: through the main admin page or through the group admin page.
With the main admin page, it should prompt you to select a group when creating the token. 
On the group page, the invite token is automatically tied to the group.

There is now also a feature that you can select a certain group as the default in the settings. Users without an invite token (if public signup is enabled) will be added to the group when they signup.

**Comment by amb1** [comment]
Thank you for the quick response, will check out the changes!
Three more things that i got as feedback after sending it out to family and friends:

- even when someone is in a group, maybe they don`t want to be gifted as well but only claim presents. therefore it could be a good idea to start off with no list but have to create one to be also ready for presents. this would also make things uncluttered in groups where a lot of people are present but not all of them want to be gifted
- would it be possible to have groups of presents, for example when someone wishes a lot of books, you have a group books in your wishlist and put all of the books in there
- the lists could be clearer if there would be a grid layout for the wishes instead of the list layout 
![4aab8bec1f85cfdc25621645c832e9d4](https://github.com/user-attachments/assets/2f72a7d4-1487-4f9d-ab2e-b1cc840a8bcd)
